enterprise-suite/scripts/lbc.py

```diff
@@ -214,11 +215,40 @@ def check_credentials(creds):
         else:
             printerr('error: check_credentials failed: {}'.format(err))
     finally:
-        return success 
+        return success
 
-def preinstall_check(creds, minikube=False, minishift=False):
+
+
+url_http_timeout=3 # 3 seconds
+installer_url="https://raw.githubusercontent.com/lightbend/console-charts/master/enterprise-suite/scripts/lbc.py"
+warning_display_timeout=2 # 2 seconds
+
+# compare the contents of current running installer and remote installer.
+# if they are different print warning a new installer is present
+def check_new_install_script():
+    try:
+        response=url.urlopen(installer_url, timeout=url_http_timeout)
```

Comment:
We had problems with Python TLS versions on Mac OS when using urllib. It can be a part of separate PR, but it's probably worth to extract http request function that tries to use curl and then urllib2 as a fallback, like what check_credentials does.